### PR TITLE
2555 by Inlead: Limit the opening hours pushed to 10 weeks from now.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/2555

### CODE REVIEW

File: modules/ding_mobilesearch/ding_mobilesearch.module [hunk 1]
```diff
@@ -353,8 +353,21 @@ function ding_mobilesearch_ding_mobilesearch_node_export_mapping($node) {
...
       ->execute();
+
+    // Push only opening hours from current week till 10 weeks ahead.
+    $current_date = date('Y-m-d', REQUEST_TIME);
+    // Align with start of the week.
+    $current_day = date('N', REQUEST_TIME) - 1;
+    $now = strtotime($current_date) - $current_day * 86400;
+    // 10 weeks.
+    $threshold = 6048000;
+
```
Comment (kasperg) [inline modules/ding_mobilesearch/ding_mobilesearch.module]: I think we can let PHP do the heavy lifting and reduce this to 

```php
strtotime('last monday + 10 weeks');
```
https://3v4l.org/Z2r2p

I think that makes it a lot clearer what the threshold is.

An explanation of why we need this threshold would also be good.

File: modules/ding_mobilesearch/ding_mobilesearch.module [hunk 2]
```diff
@@ -538,3 +551,52 @@ function theme_ding_mobilesearch_types_table(array $variables) {
 
   return $output;
 }
+
+/**
+ * Implements hook_cron().
+ */
+function ding_mobilesearch_cron() {
+  $last_cron = variable_get('ms_update_oh', 0);
```
Comment (kasperg) [inline modules/ding_mobilesearch/ding_mobilesearch.module]: Please prefix variables with the name of the responsible module. `ms_update_oh` is not very descriptive.

File: modules/ding_mobilesearch/ding_mobilesearch.module [hunk 3]
```diff
@@ -538,3 +551,52 @@ function theme_ding_mobilesearch_types_table(array $variables) {
...
+    }
+  }
+}
+
+/**
+ * Implements hook_cron_queue_info().
+ */
+function ding_mobilesearch_cron_queue_info() {
+  // Mobilesearch update opening hours.
+  $queue['ms_update_oh'] = array(
```
Comment (kasperg) [inline modules/ding_mobilesearch/ding_mobilesearch.module]: Naming the queue after the responsible module would be good as well.

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 9edbe816b44e8351bb16a5cc6d445adaed396e3c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 9edbe816b44e8351bb16a5cc6d445adaed396e3c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by kasperg** [comment]
Thanks for the update.

The failures of Scrutinizer and Circle can be disregarded here.

**Pull request merged**